src/main/java/com/eta/houzezbackend/controller/AgentController.java

```diff
@@ -40,6 +40,12 @@ public Agent activeAgent(@RequestParam String token, @PathVariable Long id) {
         return agentService.setAgentToActive(token);
     }
 
+    @PatchMapping("/reset-password")
```

Comment:
Verb in restful api?

Reply:
This is hann's code. Jessey may fix this in the future.